indra/newview/llpuppetmotion.cpp

```diff
@@ -544,29 +544,6 @@ LLIK::Constraint::ptr_t get_constraint_by_joint_id(S16 joint_id)
 // END HACK
 
 
-void LLPuppetMotion::DelayedEventQueue::addEvent(
-        Timestamp remote_timestamp,
-        Timestamp local_timestamp,
-        const LLPuppetJointEvent& event)
-{
-    if (mLastRemoteTimestamp != -1)
-    {
-        // dynamically measure mEventPeriod and mEventJitter
-        constexpr F32 DEL = 0.1f;
-        Timestamp this_period = remote_timestamp - mLastRemoteTimestamp;
-        mEventJitter = (1.0f - DEL) * mEventJitter + DEL * std::abs(mEventPeriod - (F32)this_period);
-
-        // mEventPeriod is a running average of the period between events
-        mEventPeriod = (1.0f - DEL) * mEventPeriod + DEL * this_period;
-    }
-    mLastRemoteTimestamp = remote_timestamp;
-
-    // we push event into the future so we have something to interpolate toward
-    // while we wait for the next
-    Timestamp delayed_timestamp = local_timestamp + (Timestamp)(mEventPeriod + mEventJitter);
-    mQueue.push_back({delayed_timestamp, event});
-}
```

Comment:
`LLPuppetMotion` used to both apply IK results (to the local avatar) and also play the puppetry stream (on other avatars) however the "play puppetry stream" duties have been moved to `LLStreamingMotion`.